test/test_report.py

```diff
@@ -149,10 +149,14 @@ def test_execution_summary_formatted(
     mock_popen: mock.MagicMock, mock_log_paths: mock.MagicMock
 ) -> None:
     mock_log_paths.prefix = "mock_prefix"
-    report = Report(
-        "_cmd", [], None, mock_popen, mock_log_paths, EXECUTION_SUMMARY_FORMAT, False
-    )
+    report = Report("_cmd", [], None, EXECUTION_SUMMARY_FORMAT, clobber=False)
+    # It should not crash and it would render even where no wallclock time yet
+    assert "wall clock time: nan" in report.execution_summary_formatted.lower()
```

Comment:
Do you think its safe to use this cached property this early? Previously, the cached properties weren't used until the end and the data was stable.

Reply:
good catch. Let's make them into regular property!

Reply:
done